inductiva/simulators/openfoam.py

```diff
@@ -23,19 +24,26 @@ def __init__(self, version: str = "foundation"):
         super().__init__()
         self.api_method_name = f"fvm.openfoam_{version}.run_simulation"
 
-    def run(
-        self,
-        input_dir: types.Path,
-        commands: types.Commands,
-        on: Optional[types.ComputationalResources] = None,
-        storage_dir: Optional[types.Path] = "",
-        extra_metadata: Optional[dict] = None,
-    ) -> tasks.Task:
+    @meta.deprecated_arg(n_cores="n_vcpus")
```

Comment:
This decorator should not be here, since the argument didn't exist at all for this simulator.